Dockerfile

```diff
@@ -27,7 +29,7 @@ RUN go build -a -installsuffix cgo \
 FROM busybox
 
 # =================== MAIN CONTAINER ===================
-FROM scratch as authenticator-client
+FROM alpine:3.8 as authenticator-client
```

Comment:
Thanks for the link. I actually prefer to use `alpine` because it will be needed when we make the project FIPS compliant as it has the required libraries.